### PR TITLE
[ja] Localize Immutable Infrastructure into Japanese

### LINKED ISSUE #2824: [ja] Localize Immutable Infrastructure into Japanese
Localize https://glossary.cncf.io/immutable-infrastructure/ into Japanese

### PR DESCRIPTION
**Pull request by Okabe-Junya**
### Describe your changes

Localize `immutable-infrastructure.md` into Japanese

### Related issue number or link (ex: `resolves #issue-number`)

resolves #2824 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by Okabe-Junya** [comment]
The target page on the deploy preview:

- https://deploy-preview-2857--cncfglossary.netlify.app/ja/immutable-infrastructure/

File: content/ja/immutable-infrastructure.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+イミュータブルインフラストラクチャーとは、一度デプロイされると変更することができないコンピューターインフラストラクチャー([仮想マシン](/ja/virtual-machine/)や[コンテナ](/ja/container/)、ネットワーク機器)を指します。
+これは許可されていない変更を自動的に上書きするプロセスや、最初から変更を許可しないシステムによって強制されます。
+コンテナはイミュータブルインフラストラクチャーの良い例です。
+なぜならコンテナに永続的な変更を加えるには、新しいバージョンのコンテナを作成するか、イメージから既存のコンテナを再度作成するしかないからです。
+
+許可されていない変更の防止や特定により、イミュータブルインフラストラクチャーはセキュリティリスクの特定と軽減を容易にします。
+このようなシステムの運用ははるかに簡単になります。
+なぜなら、管理者がそれについての前提を立てやすくなるためです。
+結局のところ、誰も間違いを犯していないことや、伝え忘れた変更を行っていないことが分かっています。
+イミュータブルインフラストラクチャーは[Infrastracture as Code](/ja/infrastructure-as-code/)と密接に関連しており、インフラストラクチャーを作成するために必要なすべての自動化がバージョン管理(たとえばGit)されています。
```
Comment (naonishijima) [inline content/ja/immutable-infrastructure.md:17]: typo. Infrastracture → Infrastructure

Comment (Okabe-Junya) [reply content/ja/immutable-infrastructure.md]: ありがとうございます。完全に見落としていました。。

https://github.com/cncf/glossary/pull/2857/commits/0303e3f85e8de99b49f9cb6049aa37915d19822b で修正しました

**Review by naonishijima — APPROVED** [review]
LGTM

**Review by kaitoii11 — APPROVED** [review]
LGTM

**Pull request merged**